llvm/include/llvm/CodeGen/SelectionDAGNodes.h

```diff
@@ -1006,6 +1016,7 @@ END_TWO_BYTE_PACK()
 
   SDNodeFlags getFlags() const { return Flags; }
   void setFlags(SDNodeFlags NewFlags) { Flags = NewFlags; }
+  void clearFlags(unsigned Mask) { Flags &= ~Mask; }
```

Comment:
The interface here is a little confusing now. The existence of clearFlags could make someone think that setFlags would have OR semantics.

Reply:
Do you have any suggestion on the naming? `maskOutFlags`? `dropFlags`?

Reply:
I'm ok with `dropFlags`

Reply:
Done.

Reply:
Emm... It is likely to break some downstream implementations.

Reply:
Fair enough;